src/tests/stub_libmem.cc

```diff
@@ -11,6 +11,12 @@
 #define STUB_API "mem/libmem.la"
 #include "tests/STUB.h"
 
+#include "mem/Allocator.h"
+namespace Mem {
+//Allocator::Allocator(char const *aLabel);
```

Comment:
I do not think it is a good idea to comment out inlined methods. These comments often become stale and create noise in code searches. The PR already correctly drops a similar method -- inUseCount().

```suggestion
```